### DISCUSSION TITLE
Ability to disqualified island on /is top

### DISCUSSION THREAD

**Discussion by SwartZCoding** [BG-Software-LLC/SuperiorSkyblock2#1071] [Implemented Feature Requests] [answered]
All on the title ! This command is just useful to keep is level & points but not showed on the ranking GUI.

Add command like this : 

/is admin removetop <name of a player>/<name of island> (Remove from all top type).
/is admin removetop <name of a player>/<name of island> <top type specify (optional)> (remove from a specific top type).

For sure, add a command too to readd island on top to reverse this feature :)

**Comment by OmerBenGera** [comment, marked as answer]
/is admin ignore and /is admin unigniore
